### ISSUE TITLE
Tekken 8 Update 1.01.04 Crash with Overlay

### ISSUE THREAD

**Issue by DoctorDrago513** [TekkenOverlay/TekkenOverlay#160]
Tekken 8 crashes with "UE Polaris Game Has Crashed" Fatal Error on startup with latest release of overlay running.  Crash occurs immediately after initial splash screen.

**Comment by justflores** [comment]
Same thing happens to me after updating today.

**Comment by dibnull** [comment]
yeah broken after update

**Comment by liyumyum** [comment]
same here :( the game crashes instantly after opening the overlay

**Comment by FaboDiaz** [comment]
Us potato device owners will have to wait :(

**Comment by dibnull** [comment]
i think it just need to update the memory address.

**Comment by LincolnG17** [comment]
broken

**Comment by FaboDiaz** [comment]
can someone message me when they fix it I really want to play soon and bored lol

**Comment by Paromc** [comment]
Same. I have regular crashes and flickering before, but now id doesn't work at all

**Comment by Irrumare** [comment]
Probably an easy fix, hope it will appear today.

**Comment by Hfhduewisjjs** [comment]
Same issue.

**Comment by hg1217** [comment]
Confirmed

**Comment by Irrumare** [comment]
2 days and nothing, seems like we wont be able to play the game for a while

**Comment by LashaT** [comment]
would def like a solution

**Comment by XeonTimes747** [comment]
same problem too

**Comment by FaboDiaz** [comment]
Maybe someone can DM the developers? in the off-chance they haven't noticed, going off it saying there's no "assignees" to the issue yet on the top right

**Comment by Elzeeee** [comment]
Please, give the dev some time.

**Comment by Paromc** [comment]
> Please, give the dev some time.

no one is demanding anything, people just asking for update, because there is no information whether it will happen or not

**Comment by Elzeeee** [comment]
> > Please, give the dev some time.
> 
> no one is demanding anything, people just asking for update, because there is no information whether it will happen or not

I never said anyone was demanding anything. I'm just trying to calm people's anxious hearts.
You are here and your reply also reveals you are anxious.

**Comment by Yudoken** [comment]
The struggle is real, I'm playing on a 4k/120hz/gsync tv and without the mod the lag is significantly worse.

This mod is one of my favourite, I hope we can enjoy the game without the artificial limitations soon.

**Comment by Paromc** [comment]
> The struggle is real, I'm playing on a 4k/120hz/gsync tv and without the mod the lag is significantly worse.
> 
> This mod is one of my modt favourite of all time, it fixed my favourite games biggest technical issues.
> 
> I hope we can enjoy the game without the artificial limitations soon.

you can use special k or riva tuner beta version, with reflex, input lag will decrease drastically

**Comment by CyberDNIWE** [comment]
Can confirm, I have it broken after update too, crash on startup

**Comment by FriedmarK** [comment]
same, instant crash on launch :(

**Comment by FuzzyAfro** [comment]
> > > The struggle is real, I'm playing on a 4k/120hz/gsync tv and without the mod the lag is significantly worse.
> > > This mod is one of my modt favourite of all time, it fixed my favourite games biggest technical issues.
> > > I hope we can enjoy the game without the artificial limitations soon.
> > 
> > 
> > you can use special k or riva tuner beta version, with reflex, input lag will decrease drastically
> 
> Can you expand on this? Does that allow frame generation/fps uncapping or no? Would love to find an alternative to this overlay so we don't have to keep relying on updates.

You can try Lossless Scaling on Steam, it does have frame generation and can reach 120fps. There are some very mild artifacts but overall it is great. I've been using it until this gets updated.

**Comment by Paromc** [comment]
> > > The struggle is real, I'm playing on a 4k/120hz/gsync tv and without the mod the lag is significantly worse.
> > > This mod is one of my modt favourite of all time, it fixed my favourite games biggest technical issues.
> > > I hope we can enjoy the game without the artificial limitations soon.
> > 
> > 
> > you can use special k or riva tuner beta version, with reflex, input lag will decrease drastically
> 
> Can you expand on this? Does that allow frame generation/fps uncapping or no? Would love to find an alternative to this overlay so we don't have to keep relying on updates.



> > > The struggle is real, I'm playing on a 4k/120hz/gsync tv and without the mod the lag is significantly worse.
> > > This mod is one of my modt favourite of all time, it fixed my favourite games biggest technical issues.
> > > I hope we can enjoy the game without the artificial limitations soon.
> > 
> > 
> > you can use special k or riva tuner beta version, with reflex, input lag will decrease drastically
> 
> Can you expand on this? Does that allow frame generation/fps uncapping or no? Would love to find an alternative to this overlay so we don't have to keep relying on updates.

Unfortunately don't. But maybe in the future, will be

**Comment by XeonTimes747** [comment]
does anyone know if it's bannable to be using frame uncap function in tekken 8 online?

**Comment by AndrewZ987** [comment]
> does anyone know if it's bannable to be using frame uncap function in tekken 8 online?

No, I have been using it since day 1 and I never got banned whatsoever. But for my situation, my connection is extremely bad for no reason and I randomly disconnect from the server from time to time regardless. I think it's totally irrelevant to the overlay though.

**Comment by hg1217** [comment]
Guys until the devs update it you can use "Losseless Scaling" from steam.
enable frame generation and you get 120fps.

**Comment by dibnull** [comment]
> > Guys until the devs update it you can use "Losseless Scaling" from steam store. enable frame generation and you get 120fps.
> 
> I don't think that actually lowers the input lag/truly uncaps the framerate, it's just a visual fps thing. It's still gonna have the same input lag as 60fps iirc.

yeah it's a fake fps. Why can't just BANDAI NAMCO enable this feature on their game for pc unlock that stupid 60fps.

**Comment by CyberDNIWE** [comment]
> yeah it's a fake fps. Why can't just BANDAI NAMCO enable this feature on their game for pc unlock that stupid 60fps.

Because PS5 cant handle 120 fps and with Sony being their sponsor in more ways than one, they cant make PC version **that much better**.
_Corpo politics ruins enjoyment once again... typical corpos..._

**Comment by AniMittra** [comment]
> > > > The struggle is real, I'm playing on a 4k/120hz/gsync tv and without the mod the lag is significantly worse.
> > > > This mod is one of my modt favourite of all time, it fixed my favourite games biggest technical issues.
> > > > I hope we can enjoy the game without the artificial limitations soon.
> > > 
> > > 
> > > you can use special k or riva tuner beta version, with reflex, input lag will decrease drastically
> > 
> > 
> > Can you expand on this? Does that allow frame generation/fps uncapping or no? Would love to find an alternative to this overlay so we don't have to keep relying on updates.
> 
> You can try Lossless Scaling on Steam, it does have frame generation and can reach 120fps. There are some very mild artifacts but overall it is great. I've been using it until this gets updated.

Wouldn't Lossless Scaling add input lag? I don't think it's a good solution for a fighting game. Probably fine for single player games.

**Comment by Paromc** [comment]
> I wanna to uncapped fps for the game so I can undervolt easily hopefully to fix stuttters I'm having and heating issues with my laptop.
> 
> Is there any good alternatives, even outside of T8 assuming there is not other FPS uncap solution? :(

You can undervolt without performance decrease. Also, you tried potato stages mods?

**Comment by PetriWW** [comment]
> I wanna to uncapped fps for the game so I can undervolt easily hopefully to fix stuttters I'm having and heating issues with my laptop.
> 
> Is there any good alternatives, even outside of T8 assuming there is not other FPS uncap solution? :(

Lossless Scaling program can make the game 120fps

**Comment by Mackzim** [comment]
Please fix this. UE error here aswell.

**Comment by wonbin7975** [comment]
dev abandoned project. we have to wait untill other dev recreate

**Comment by Mackzim** [comment]
> dev abandoned project. we have to wait untill other dev recreate

I call cap

**Comment by lamovnik** [comment]
Ther's going to be Tekken update on Tuesday 20., so the Overlay devs are waiting it out probably. And there will be update after that.

**Comment by weizhen199** [comment]
> cap

in game announcement

**Comment by weizhen199** [comment]
> > There's going to be Tekken update on Tuesday 20., so the Overlay devs are probably just waiting it out. And there will be update after that...
> > right?
> 
> Where are you guys getting these infos from? I tried finding the Dev on X but no results.

announcement

**Comment by GotekSC** [comment]
> fatal error on my end, but when i try to run a stage, i cant even load anything anymore.
> 
> ran the overlay with administrator permissions, and it injected the dll, then crashes. every single time ;-;
> 
> i was playing better before. i wanna turn everything back, but i've reinstalled 3 times and it still didnt give me the base game. does anyone know a way to reverse everything back to normal? steam cloud disabling didnt help either

1. turn off steam cloud syncing for Tekken 8.
2. Go to the directories mentioned in `Configuration file(s)` and `Save game data location`.
3. Launch game, recreate all your settings, test a bit, exit game.
4. Re-enable steam cloud syncing for Tekken 8.

**Comment by oOiKonGiOo** [comment]
> > fatal error on my end, but when i try to run a stage, i cant even load anything anymore.
> > ran the overlay with administrator permissions, and it injected the dll, then crashes. every single time ;-;
> > i was playing better before. i wanna turn everything back, but i've reinstalled 3 times and it still didnt give me the base game. does anyone know a way to reverse everything back to normal? steam cloud disabling didnt help either
> 
> 1. Turn off steam cloud syncing for Tekken 8.
> 2. Go to the directories mentioned in `Configuration file(s)` and `Save game data location` from this [article](https://www.pcgamingwiki.com/wiki/Tekken_8#Configuration_file.28s.29_location). Make a backup of these directories, and delete the originals.
> 3. Launch game, recreate all your settings, test a bit, exit game.
> 4. Re-enable steam cloud syncing for Tekken 8.

anybody tested this yet?

**Comment by GotekSC** [comment]
> > > fatal error on my end, but when i try to run a stage, i cant even load anything anymore.
> > > ran the overlay with administrator permissions, and it injected the dll, then crashes. every single time ;-;
> > > i was playing better before. i wanna turn everything back, but i've reinstalled 3 times and it still didnt give me the base game. does anyone know a way to reverse everything back to normal? steam cloud disabling didnt help either
> > 
> > 
> > 
> > 1. Turn off steam cloud syncing for Tekken 8.
> > 2. Go to the directories mentioned in `Configuration file(s)` and `Save game data location` from this [article](https://www.pcgamingwiki.com/wiki/Tekken_8#Configuration_file.28s.29_location). Make a backup of these directories, and delete the originals.
> > 3. Launch game, recreate all your settings, test a bit, exit game.
> > 4. Re-enable steam cloud syncing for Tekken 8.
> 
> anybody tested this yet?

that was just a reply to the guy that wanted to revert to the default state. I don't know how just not running the overlay didnt help, but in the case that his settings were messed up thats how to reset everything and have steam cloud stop from re-syncing the settings in the case that it is the culprit

that had nothing to do with *running* the tekken overlay and getting it to work

**Comment by rllanger** [comment]
Works in new release
https://github.com/TekkenOverlay/TekkenOverlay/releases/tag/7.5.11-8.1.01.04-0

**Comment by wonbin7975** [comment]
> d

finally

**Comment by maylique** [comment]
i dont get it, it worked for 1 session. now i get the crash again.

**Comment by rllanger** [comment]
> Works in new release https://github.com/TekkenOverlay/TekkenOverlay/releases/tag/7.5.11-8.1.01.04-0

Actually it still crashes for me

**Comment by dibnull** [comment]
> > Works in new release https://github.com/TekkenOverlay/TekkenOverlay/releases/tag/7.5.11-8.1.01.04-0
> 
> Actually it still crashes for me

make sure there are no other overlay programs running (e.g msi afterburn)

**Comment by CyberDNIWE** [comment]
> Issue is not fully resolved, the game now gets as far as the Start screen but crashes with the same error as before.

Remove the last version properly, including all configs etc. Disable additional overlays (like RivaTuner, Reshade, Steam etc) or wait for them to pop up during the intro videos.
It crashed on me first two tries, but after I've disabled all of the above it ran just fine. (Same with the previous version AND T7 version by the way).

**Comment by DoctorDrago513** [comment]
> > Issue is not fully resolved, the game now gets as far as the Start screen but crashes with the same error as before.
> 
> Remove the last version properly, including all configs etc. Disable additional overlays (like RivaTuner, Reshade, Steam etc) or wait for them to pop up during the intro videos. It crashed on me first two tries, but after I've disabled all of the above it ran just fine. (Same with the previous version AND T7 version by the way).

Still no luck - Steam, Rivatuner and Afterburner disabled with a completely fresh Tekken Overlay install still cant get past the opening cinematic.

**Comment by lamovnik** [comment]
> > > Issue is not fully resolved, the game now gets as far as the Start screen but crashes with the same error as before.
> > 
> > 
> > Remove the last version properly, including all configs etc. Disable additional overlays (like RivaTuner, Reshade, Steam etc) or wait for them to pop up during the intro videos. It crashed on me first two tries, but after I've disabled all of the above it ran just fine. (Same with the previous version AND T7 version by the way).
> 
> Still no luck - Steam, Rivatuner and Afterburner disabled with a completely fresh Tekken Overlay install still cant get past the opening cinematic.

Rivatuner is no issue, if there is no global or per game frame limiter active and you don't open the overlay info while in-game. Afterburner is also OK to be running. Steam overlay should be no issue as well, although I think it's better to have it off regardeless do to performance costs and other issues.

What you need to do is run the the Overlay program as an Administrator. If you don't want to do this every time, right click it - Compatibility - Run this program as an Administrator.

**Comment by DoctorDrago513** [comment]
> > > > Issue is not fully resolved, the game now gets as far as the Start screen but crashes with the same error as before.
> > > 
> > > 
> > > Remove the last version properly, including all configs etc. Disable additional overlays (like RivaTuner, Reshade, Steam etc) or wait for them to pop up during the intro videos. It crashed on me first two tries, but after I've disabled all of the above it ran just fine. (Same with the previous version AND T7 version by the way).
> > 
> > 
> > Still no luck - Steam, Rivatuner and Afterburner disabled with a completely fresh Tekken Overlay install still cant get past the opening cinematic.
> 
> Rivatuner is no issue, if there is no global or per game frame limiter active and you don't open the overlay info while in-game. Afterburner is also OK to be running. Steam overlay should be no issue as well, although I think it's better to have it off regardeless due to performance cost and other issues.
> 
> What you need to do is run the the Overlay program as an Administrator. If you don't want to do this every time, right click it - Compatibility - Run this program as an Administrator.

I've had Tekken Overlay set to run as administrator from the start.

**Comment by lamovnik** [comment]
> > > > > Issue is not fully resolved, the game now gets as far as the Start screen but crashes with the same error as before.
> > > > 
> > > > 
> > > > Remove the last version properly, including all configs etc. Disable additional overlays (like RivaTuner, Reshade, Steam etc) or wait for them to pop up during the intro videos. It crashed on me first two tries, but after I've disabled all of the above it ran just fine. (Same with the previous version AND T7 version by the way).
> > > 
> > > 
> > > Still no luck - Steam, Rivatuner and Afterburner disabled with a completely fresh Tekken Overlay install still cant get past the opening cinematic.
> > 
> > 
> > Rivatuner is no issue, if there is no global or per game frame limiter active and you don't open the overlay info while in-game. Afterburner is also OK to be running. Steam overlay should be no issue as well, although I think it's better to have it off regardeless due to performance cost and other issues.
> > What you need to do is run the the Overlay program as an Administrator. If you don't want to do this every time, right click it - Compatibility - Run this program as an Administrator.
> 
> I've had Tekken Overlay set to run as administrator from the start.

Well then, it's some fuckery beyond my experience. Good luck to you!

**Comment by libboj** [comment]
Still doesn't work, fatal error or DXGI_ERROR_DEVICE_HUNG.

**Comment by Paromc** [comment]
it works, but crash a lot

**Comment by AniMittra** [comment]
I'm still getting the "UE Polaris" crash. Newest release, NVIDIA 3060 Ti, Steam and other overylays off. 240hz 1440p monitor.

First time I tried it worked up through the main menu, but when I went to practice and selected characters, it crashed during the following loading screen. Every try after that has been an immediate crash.

FWIW I had tweaked the "Engine.ini" file to make some custom graphics adjustments (motion blur, bloom, etc), could this be causing an issue?

**Comment by HellSweep69** [comment]
It still crashes as soon as the cinematic ends. Polaris Fatal Error.

**Comment by HellSweep69** [comment]
Now game works with overlay open but overlay doesn't actually work. App doesn't give any errors but has also no effect in game.
